test/spec/utils/Options.spec.ts

```diff
@@ -18,181 +18,168 @@ describe('n3Charts.Utils.Options', () => {
 
       expect(testing).to.equal(expected);
     });
-  });
 
-  describe('parse', () => {
-    beforeEach(() => {
+    it('should create a series property with the type array', () => {
       options = new Options();
-    });
-
-    it('should call parseSeries function with series property', () => {
-      var arg = { series: ['foo', 'bar'] };
-      var spy = sinon.spy(options, 'parseSeries');
 
-      options.parse(arg);
-
-      var testing = spy.calledWith(arg.series);
+      var testing = angular.isArray(options.series);
       var expected = true;
 
       expect(testing).to.equal(expected);
     });
 
-    it('should call parseSeries function with default arguments', () => {
-      var arg = { foo: 'bar' };
-      var spy = sinon.spy(options, 'parseSeries');
-
-      options.parse(arg);
+    it('should create a margin property with the type object', () => {
+      options = new Options();
 
-      var testing = spy.calledWith(Options.DEFAULT.series);
+      var testing = angular.isObject(options.margin);
       var expected = true;
 
       expect(testing).to.equal(expected);
```

Comment:
Ok, then we agree on a more flexible structure. However, I will try to continue writing tests in this format - and we should decide later on strict testing guidelines (and write them down)